packages/blockchain-environment-v1/src/FabricCertificateAuthority.ts

```diff
@@ -24,13 +22,19 @@ export class FabricCertificateAuthority implements IFabricCertificateAuthority {
 
     private static _instance: FabricCertificateAuthority = new FabricCertificateAuthority();
 
-    public async enroll(caUrl: string, enrollmentID: string, enrollmentSecret: string, caName?: string): Promise<{certificate: string, privateKey: string}> {
-        const outputAdapter: VSCodeBlockchainOutputAdapter = VSCodeBlockchainOutputAdapter.instance();
+    public async enroll(caUrl: string, enrollmentID: string, enrollmentSecret: string, caName?: string, _outputAdapter?: OutputAdapter): Promise<{ certificate: string, privateKey: string }> {
```

Comment:
By renaming `_outputAdapter` to `outputAdapter`, and removing `let outputAdapter: OutputAdapter;`, we could just do:

```
if(!outputAdapter){
    outputAdapter = ConsoleOutputAdapter.instance();
}
```